webapp/channels/src/components/global_header/right_controls/saved_posts_button/saved_posts_button.tsx

```diff
@@ -31,34 +29,31 @@ const SavedPostsButton = (): JSX.Element | null => {
         }
     };
 
-    const tooltip = (
-        <Tooltip id='recentMentions'>
-            <FormattedMessage
-                id='channel_header.flagged'
-                defaultMessage='Saved messages'
-            />
-        </Tooltip>
+    const icon = (
+        <IconButton
+            size={'sm'}
+            icon={'bookmark-outline'}
+            toggled={rhsState === RHSStates.FLAG}
+            onClick={savedPostsButtonClick}
+            inverted={true}
+            compact={true}
+            aria-expanded={rhsState === RHSStates.FLAG}
+            aria-controls='searchContainer' // Must be changed if the ID of the container changes
+            aria-label={formatMessage({id: 'channel_header.flagged', defaultMessage: 'Saved messages'})}
+        />
```

Comment:
please move this under with WithTooltip.

Reply:
Hey @M-ZubairAhmed , addressed the above!